package.json

```diff
@@ -76,24 +77,18 @@
     "rollup-plugin-node-resolve": "^5.0.2",
     "stylelint": "^10.1.0",
     "stylelint-config-standard": "^18.3.0",
-    "testem": "^2.16.0",
-    "vendor-copy": "^2.0.0"
+    "testem": "^2.16.0"
   },
-  "esmify": [
-    "form-serialize",
-    "camelcase-keys",
-    "lodash.kebabcase",
-    "pwa-helpers/media-query",
-    "currency-symbol-map"
-  ],
-  "vendorCopy": [
-    {
-      "from": "node_modules/lit-html",
-      "to": "web_modules/lit-html"
-    },
-    {
-      "from": "node_modules/@ubio/client-library/index.js",
-      "to": "web_modules/@ubio/client-library.js"
-    }
-  ]
+  "@pika/web": {
+    "webDependencies": [
+      "@ubio/client-library",
+      "form-serialize",
+      "camelcase-keys",
+      "lodash.kebabcase/index.js",
```

Comment:
It's not common for modules on npm to have dots in their names (I think it's not even possible to do for new modules in the same way as capital letters are banned). `pika` has a bug with handling them so the index file is referenced directly here. The change is reflected in the two modules which use it.